amf-client/shared/src/test/resources/upanddown/cycle/raml10/multiple-media-types/api.raml.expanded.jsonld

```diff
@@ -1164,7 +1164,30 @@
             ],
             "http://a.ml/vocabularies/core#mediaType": [
               {
-                "@value": "application/raml+yaml"
+                "@value": "application/yaml"
```

Comment:
Raml10CycleTestByDirectory: this is a raml api that references and external yaml file, previously the external yaml file did not match with any parsing plugin and was handled as an ExternalFragment. Now the ExternalJsonYamlRefsPlugin is selected as a parsing plugin and return and external fragment with annotations. (dont see any risk in this change)